deb/build.sh

```diff
@@ -75,16 +75,31 @@ tar -xvf "$JJS_TGZ"
 mv jjs pkg
 
 mkdir data
```

Comment:
Now useless?

Reply:
No

Reply:
```suggestion
mkdir -p data/opt
mv jjs data/opt/jjs
mkdir -p data/lib/systtemd/system
```

Reply:
I think this is semantically incorrect, since data will also contain other directories

Reply:
For example?

Reply:
All system directories with symlinks to /opt/jjs